site/passwords.xml

```diff
@@ -209,8 +209,60 @@ credential_validator.regexp = ^[a-bA-Z0-9$]{20,100}
       </doc:subsection>
     </doc:section>
 
+    <doc:section name="passwordless-users">
+      <doc:heading>Passwordless Users</doc:heading>
+      <p>
+        <a href="/access-control.html">Internal authentication backend</a> allows for users without a password
+        or with a blank one (assuming credential validator also allows it). Such users are only mean to be used
+        with passwordless <a href="/authentication.html">authentication mechanisms</a> such as <a href="https://github.com/rabbitmq/rabbitmq-auth-mechanism-ssl">authentication using x509 certificates</a>.
+      </p>
+
+      <p>
+        In order to create a passwordless user, create one with any password that passes validation and clear
+        the password using <a href="/cli.html">rabbitmqctl</a>'s <code>clear_password</code> command:
+
+        <pre class="sourcecode bash">
+rabbitmqctl add_user passwordless-user "pa$$wordless"
+rabbitmqctl clear_password passwordless-user
+# don't forget to grant the user virtual host access permissions using set_permissions
+# ...
+        </pre>
+      </p>
+
+      <p>
+        Starting with version <code>3.7.3</code>, authentication attempts that use a blank password
+        will be unconditionally rejected by the <a href="/access-control.html">internal authentication backend</a> with a distinctive error
+        message in the <a href="/logging.html">server log</a>. Connections that authenticate using x509 certificates or use an external service
+        for authentication (e.g. <a href="/ldap.html">LDAP</a>) can use blank passwords.
+      </p>
+    </doc:section>
+
+    <doc:section name="x509-certificate-authentication">
+      <doc:heading>Authentication Using TLS (x509) Certificates</doc:heading>
+      <p>
+        It is possible to <a href="https://github.com/rabbitmq/rabbitmq-auth-mechanism-ssl">authenticate connections using x509 certificates</a> and avoid
+        using passwords entirely. The authentication process then will rely on TLS <a href="https://tools.ietf.org/html/rfc5280#section-6">peer certificate chain validation</a>.
+      </p>
+
+      <p>
+        To do so:
+
+        <ul>
+          <li>Create a passwordless user (see above)</li>
+          <li>Enable the <a href="https://github.com/rabbitmq/rabbitmq-auth-mechanism-ssl">rabbitmq-auth-mechanism-ssl</a> plugin</li>
+          <li>Follow the plugin's configuration instructions</li>
+          <li>Configure client connections to use TLS and the <code>EXTERNAL</code>  authentication mechanism</li>
```

Comment:
There is an extra whitespace after `</code>`. Also ... is there a document explaining this `EXTERNAL` authentication mechanism and clients? I found [this PR](https://github.com/streadway/amqp/pull/121), for instance. Should this be called `SASL EXTERNAL` instead?

Reply:
`EXTERNAL` is the value provided by clients and used in the spec. There is a guide on [authentication mechanisms](http://www.rabbitmq.com/authentication.html) which I will expand to cover what's currently in [rabbitmq-auth-mechanism-ssl](https://github.com/rabbitmq/rabbitmq-auth-mechanism-ssl)'s README.